examples/acquiring_data/goes_xrs_example.py

```diff
@@ -86,10 +81,7 @@
 # flares.
 
 goes_flare = goes_15.truncate("2015-06-21 09:35", "2015-06-21 10:30")
-fig, ax = plt.subplots()
-ax.plot(goes_flare.index, np.gradient(goes_flare.quantity("xrsb")))
```

Comment:
this is plotting the derivative here not just the flare so `goes_flare.plot()` isn't the same thing.

Reply:
Oh I did not notice that.

Reply:
Fixed.